projects/example-app-cypress/integration/round-trip.spec.ts

```diff
@@ -1,14 +1,34 @@
+let LOCAL_STORAGE_MEMORY = {};
+
+Cypress.Commands.add('saveLocalStorage', () => {
```

Comment:
Add these to the commands at https://github.com/ngrx/platform/blob/master/projects/example-app-cypress/support/index.js